curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sum-all-numbers-in-a-range.english.md

```diff
@@ -8,8 +8,17 @@ challengeType: 5
 ## Description
 <section id='description'>
 We'll pass you an array of two numbers. Return the sum of those two numbers plus the sum of all the numbers between them.
-The lowest number will not always come first.
-Remember to use <a href='http://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck/19514' target='_blank'>Read-Search-Ask</a> if you get stuck. Try to pair program. Write your own code.
+The lowest number will not always come first.Remember to use <a href='http://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck/19514' target='_blank'>Read-Search-Ask</a> if you get stuck. Try to pair program. Write your own code.
```

Comment:
Please add a space between the first sentence and the second sentence.  Also, in your example, add a space between the comma and the second item in the array.  I also feel if you are going to add an example, it should show the first element as the larger number to highlight the point of the second sentence.